sources/console.c

```diff
@@ -865,54 +882,99 @@ od_console_query_set_storage_state(od_client_t *client, od_parser_t *parser,
     od_token_t token;
     int rc = od_parser_next(parser, &token);
 
-    char *state_name = state == OD_STORAGE_PAUSE ? "PAUSED" : "RESUME";
+    char state_name[] = "PAUSED";
 
     if (rc == OD_PARSER_KEYWORD)
     {
         storage_name = token.value.string.pointer;
         od_log(&instance->logger, "console", client, NULL,
-                "Making storage %s %s...", token.value.string.pointer, state_name);
+                "Making storage %.*s %s...", token.value.string.size, token.value.string.pointer, state_name);
     }
     else
```

Comment:
Parsing logic here is a slightly incomplete and have a room for errors. This should expect PARSER_EOF token, otherwise it is possible to pass anything like `PAUSE 1234` and it will treat it as pause all condition.

Also I suggest to follow style convention and start log messages from lowercase letter and remove `...`.